graphql-proxy/src/schema/resolvers.ts

```diff
@@ -0,0 +1,7 @@
+import { merge } from "lodash";
```

Comment:
I think this imports all of lodash, import instead just the merge function from `lodash/merge`

Reply:
Changed